text.cpp

```diff
@@ -396,5 +396,70 @@ end)";
 	THREAD_LINK_FROM = u8"Номер нити, от которой связывать";
 	THREAD_LINK_TO = u8"Номер нити, к которой привязывать";
 #endif // RUSSIAN
+
+#ifdef INDONESIAN
+	ATTACH = u8"Tempelkan kedalam game";
+	LAUNCH = u8"Mulai game";
+	DETACH = u8"Lepaskan dari game";
+	ADD_HOOK = u8"Tambahkan hook";
+	SAVE_HOOKS = u8"Simpan hook";
+	SETTINGS = u8"Pengaturan";
+	EXTENSIONS = u8"Ekstensi";
+	SELECT_PROCESS = u8"Pilih Proses";
+	ATTACH_INFO = u8R"(Jika kamu tidak dapat melihat proses yang akan ditempelkan, coba menjalankan dengan mode administrator
+	Kamu juga dapat mengetik process id game yang akan ditempel)";
+	SEARCH_GAME = u8"Pilih dari komputer";
+	PROCESSES = u8"Proses (*.exe)";
+	SAVE_SETTINGS = u8"Simpan pengaturan";
+	EXTEN_WINDOW_INSTRUCTIONS = u8R"(Drag and drop file ekstensi (.dll) kedalam sini dari komputer kamu untuk menambah ekstensi
+(Tidak bekerja dalam mode administrator)";
```

Comment:
Seems to be missing instructions on how to remove and reorder extensions?

Reply:
ah yes i forget that one, done fixing it. please check it out